### ISSUE TITLE
Why Was Defender Scan Terminated?

### ISSUE THREAD

**Issue by bsfinkel** [MicrosoftDocs/microsoft-365-docs#7187]
I received a Defender EventID 1002 during a full scan. All this documentation tells me is that the scan was terminated before it completed. I was not at the computer when the scan terminated, so I was not the one who terminated the scan. I want more information as to why the full Defender scan was terminated. This is Windows 10 professional 21H2 64-bit. Thanks.

In my previous (closed) posting on this topic yogkumgit replied "@msbemba Thank you very much for the contribution and sharing this explanation. @bsfinkel Considering the nature of the issue, as advised, please open a service ticket in your tenant and follow with them for the resolution."  I do not understand "open a service ticket in your tenant"; what is "tenant"?

**Comment by msbemba** [comment]
@bsfinkel Thank you for your feedback. To investigate and resolve your issue, we recommend you contact the support team so a dedicated engineer can work with you to identify and resolve the problem.

If you are Microsoft 365 Customer you get a tenant and the administrators have access to open a support ticket. If You are not a Microsoft 365 Customer, you can see the option https://support.microsoft.com/contactus

Hope this helps!

Thanks
Sri

**Comment by bsfinkel** [comment]
I prefer accessing any support group; via e-mail or the web, where I can clearly state my problem and then have the technical staff route my concern to the appropriate support personnel.  I went to the "contact us" URL and opened a chat session.  The first technician realized that I needed someone more advanced, so I was transferred to ... someone in Billing and Accounting.  I was then transferred to The Windows Technical Team.  I waited for an hour, and no one else responded.  I tried again last night at 8:00, and I left the computer at 10:00 after no one had responded.  When I got back to my computer at midnight, I saw that someone had responded at 10:47 and wondered where I was.  I will try another chat tomorrow during the day; maybe I will have better luck then.
